docs/Data Lake.md

```diff
@@ -154,6 +155,85 @@ Finally, on clicking the *Execute Rules* button, you will be able to get a preli
 
 ![Preliminary hitlist](../img/Discover/image13.png)
 
+##DepMap CCLE
+
+###Opening the app
+
+Upon opening the Discover portal on Polly, choose the Depmap data repository. The page should look something like this.
+
+![Selecting app](../img/DepMapCCLE/image1.png)
+
+After selecting the Depmap repository, you’ll be able to see a dashboard and the DepMap CCLE app icon. The dashboard will give you an overview of the repository, for instance, the information of different cell lines in the  repository, distribution of tissue lineages, metastatic status of samples, etc. among the datasets.
+
+![Kibana dashboard](../img/DepMapCCLE/image2.png)
+
+Click on the below icon to start the DepMap CCLE app.
+
+![DepMap icon](../img/DepMapCCLE/image3.png)
+
+The app will open and you should see the home page which contains two examples describing in brief  usage of the application.
+
+![App overview](../img/DepMapCCLE/image4.png)
+
+###Exploring the DepMap CCLE data lake
+
+The app provides the functionality to explore the DepMap CCLE data lake. We can query the datalake for three different functionalities for different genes in the application. Each functionality has a separate tab in left for easy access. 
+
+*   Gene Essentiality
+
+*   Gene Cluster
+
+*   CCLE Heatmap
+
+####Gene Essentiality 
+
+Gene Essentiality of the gene describes the efficacy vs selectivity map for a gene based on different models. The terms can be defined as: 
+
+*   **Efficacy** : how essential the gene is in the sensitive cell lines (the more negative the efficacy is, the more essential the gene is).
+
+*   **Selectivity** : how selectively essential the gene is between sensitive and resistant cell lines (the more positive the selectivity is, the more selective the gene is).
+
+As shown in the following image, one can search for a gene for it’s essentiality and press Go to check the scores. 
+
+![Select gene](../img/DepMapCCLE/image5.png)
+
+The search returns a Essentiality map in the left with the searched gene highlighted with Red. The right panel describes the efficacy and selectivity scores of the matched gene and the dependency scores in different models in the datalake. 
```

Comment:
*an essentiality map (Italics).....*on the left *panel............... with *red color. 
..of the matched genes(italics) and dependency scores(Italics)
...scores *of different models

Reply:
changed